webapp/src/App.js

```diff
@@ -78,53 +84,63 @@ class DecksList extends React.Component {
 
 class DecksListDeck extends React.Component {
   proptypes = {
-    deck: PropTypes.object,
-    api: PropTypes.instanceOf(Api),
+    deck: PropTypes.object.isRequired,
+    deleteDeck: PropTypes.func.isRequired,
```

Comment:
Adding the `isRequired` simply generates a warning in the JS console when the component is mounted without that prop being set to make it easier to spot bugs.